wix/Lang/OpenJDK.Base.en-us.wxl

```diff
@@ -18,4 +18,8 @@
 	<String Id="DowngradeDenied">"A later version of [ProductName] is already installed. Setup will now exit."</String>
 	<String Id="FeatureOracleJavaSoftName">JavaSoft (Oracle) registry keys</String>
 	<String Id="FeatureOracleJavaSoftDescription">Overwrites the reg keys HKLM\Software\JavaSoft (Oracle). After uninstallation of AdoptOpenJDK, Oracle Java running from PATH "C:\Program Files (x86)\Common Files\Oracle\Java\javapath" does not work. You need to reinstall it to recreate these registry keys</String>
+	<String Id="FeatureIcedTeaWebTitle">IcedTea-Web</String>
+	<String Id="FeatureIcedTeaWebDescription">Install the IcedTea-Web package</String>
```

Comment:
While we know this feature by the internal project codename 'IcedTea-Web",  I think it may be better to refer to the install option in the UI by the technology name "JNLP".  WDYT?

Reply:
Note intent to move the user doc for installers to the website here: https://github.com/AdoptOpenJDK/openjdk-website/pull/479 Still some work to do if we are using a new INSTALLLEVEL and changing the name :-/

Reply:
What is the status here please? 

- Are we changing the name from IcedTea-Web to JNLP?
- Are we creating a new INSTALLLEVEL=3 for IcedTea-Web or will the following features be part of INSTALLLEVEL=2: FeatureIcedTeaWeb,FeatureJNLPFileRunWith

@tellison @charliegracie @karianna @douph1 ?? Anyone else?

Reply:
I'm not close enough to the technology from an end user perspective.  I'd argue that we should follow the experience that the user has today with the existing Oracle Java / Java Webstart and/or IcedTeaWeb installation.  Do we have any end users who can check those settings?

Reply:
End user here.  The Oracle Java installer from java.com has java webstart functionality built in and default, which I use for connecting to my Dell DRAC out of band server management.  No separate installs or special options to choose in the Oracle installer.  jnlp files are automatically associated.  I suspect java webstart is rarely used outside of sysadmin circles these days, so I think a separate INSTALLLEVEL is fine.  If you want it to match the Oracle installer, though, IcedTea-Web and .jnlp file association should be default.

Reply:
> * As an optional install the new feature should be available in a new INSTALLLEVEL=3

This would mean that selecting INSTALLLEVEL=3 would install all other optionnal features JAVA_HOME/JavaSoft key/PATH.. and maybe we dont want all juste because we want ITW.

I think INSTALLLEVEL=1 is for JDK/JRE ( required )
and all other are LEVEL=2
As ITW don't depend on JAVA_HOME or PATH or  JavaSoft they must not be greater than 2.

INSTALLLEVEL=1 is minimal
INSTALLLEVEL=2 is all optionnal ( Full install )

If you need finer grained install use ADDLOCAL with list of features

Reply:
>I suspect java webstart is rarely used outside of sysadmin circles these days

I dont kniw what "rarely" is but
my company ( and probably many others ) are selling sofware available by JavaWebStart to many end-user ( more than 8000 uniq user this last month ) and this is not sysadmin software

Reply:
I am happy with it just being an ADDLOCAL option and not a new INSTALLLEVEL.  Really I just want unattended install capability, which an ADDLOCAL option for ITW will give me just like the current version does with setting the JAVA_HOME variable. 
No offense was meant with the "rarely used" comment, I was just speaking to my personal experience.  I thought it was a deprecated technology, I didn't know it was still in active use.

Reply:
Our company uses it for sysadmin **and** we sell software to customers that uses it. As far as Oracle is concerned, it's deprecated, but I suspect quite a lot of enterprises that use/sell Java Desktop software use it. Ideally I would prefer it installed by default, but not a huge problem if we have to document a "tick this box" - I understand that not everybody necessarily wants it installed.

Reply:
It would be good to get some closure here. My vote is with @douph1:

>INSTALLLEVEL=1 is minimal
>INSTALLLEVEL=2 is all optionnal ( Full install )

>If you need finer grained install use ADDLOCAL with list of features


Who else is voting? ;-)
Would there be new changes needed to implement this?